internal/xfn/required_resources.go

```diff
@@ -181,7 +182,13 @@ func (e *ExistingRequiredResourcesFetcher) Fetch(ctx context.Context, rs *fnv1.R
 			return nil, errors.Wrap(err, "cannot list required resources")
 		}
 
+		// Sort items by resource name so that the order is stable across calls.
+		sort.Slice(list.Items, func(i, j int) bool {
+			return list.Items[i].GetName() < list.Items[j].GetName()
+		})
```

Comment:
Should we do this in render too, for consistency?

Reply:
Yes, definitely! Good catch.

Reply:
in render we doing it like this - or did we removed this ? https://github.com/crossplane/crossplane/pull/5030